solr/CHANGES.txt

```diff
@@ -39,6 +39,8 @@ Improvements
 
 * SOLR-17685: Simplify bin/solr scripts by no longer SOLR_TOOL_HOST variable in favour of existing SOLR_HOST ENV variable. (Eric Pugh, Houston Putnam)
```

Comment:
Nooo my name, `Putman` not `Putman`. I can fix that separately 😉

Reply:
i think there is a typo in your comment on how to spell your name!   (or I can not read!)